### ISSUE TITLE
Make documentation sites more readable

### ISSUE THREAD

**Issue by OnkelTem** [yargs/yargs#2176]
In my humble opinion, the left column is very hard to read:

![image](https://user-images.githubusercontent.com/114060/166918178-a0b7e5ae-50ea-41a5-85ef-043d4941a92a.png)

It's too narrow, so that every other line gets wrapped, which makes it really hard to read, and it cannot be resized, yet at regular 1980px width:
- it allocates just 11% of the width
- right column takes another 40%
- so ~50% of the width is unused.

Please, do something. Or I can do it, if you say where can I find the site sources.

**Comment by OnkelTem** [comment]
Good catch! Thank you @khill-fbmc 

Have you published it already?

**Comment by kevinkhill** [comment]
Where is a good place to publish?

**Comment by OnkelTem** [comment]
> Where is a good place to publish?

Oh, I thought you know. Well, I take the default place where Violentmonkey/Tampermonkey extensions look for user scripts is this repository: https://greasyfork.org/

To publish your user script, you can go here https://greasyfork.org/en/scripts/by-site/yargs.js.org and click the "[Publish a script you've written](https://greasyfork.org/en/script_versions/new)"

They'll want you to register though.

**Comment by kevinkhill** [comment]
😂 I just make user scripts to fix my own annoyances. I never thought to publish, lol

I'll look into it. Are you using it? Did it work for you?

**Comment by kevinkhill** [comment]
Easy enough 😃 https://greasyfork.org/en/scripts/446661-yargs-docs-fix

**Comment by OnkelTem** [comment]
Yes, thank you! Successfuly installed the script and it worked :)

**Comment by kevinkhill** [comment]
Awesome! What a productive 24hrs we just had 👍🏻

**Comment by OnkelTem** [comment]
> Awesome! What a productive 24hrs we just had 👍🏻

Ahahah. True, bro!
